code-challenges.test.js

```diff
@@ -11,40 +11,215 @@
+*/
+
+//       I had to go to "object land" to get these values
+//                  v       v
+// const tally = ({upVotes, downVotes}) => {
+    
+//     return upVotes - downVotes
+// }
+
+// console.log(tally(votes1));  did this to test the function 
+
+/*
+ PASS  ./code-challenges.test.js
+  tally
+    ✓ takes in an object that contains up votes and down votes and returns the end tally. (1 ms)
+    ✓ takes in an object that contains up votes and down votes and returns the end tally.
+
+Test Suites: 1 passed, 1 total
+Tests:       2 passed, 2 total
+Snapshots:   0 total
+Time:        0.193 s, estimated 1 s
+Ran all test suites.
+✨  Done in 1.33s.
+
+*/
+
+
+
+
+
+
+
 
 // --------------------3) Create a function that takes in two arrays as arguments and returns one array with no duplicate values. STRETCH: Use the spread operator to pass in a dynamic number of arguments.
 
+/*
+describe statement will be something relevant like "combined" idk I'm running out of brain juice at the moment, going a little crazy coding 👹
+Only 1 it statement as we will join the arrays together 
+input: 
+const dataArray1 = ["array", "object", "number", "string", "Boolean"]
+const dataArray2 = ["string", "null", "Boolean", "string", "undefined"]
+output: 
+["array", "object", "number", "string", "Boolean", "null", "undefined"]
+I will focus to the technicallities of removing duplicates when it comes to coding out the function
+*/
+
+
 // a) Create a test with an expect statement using the variables provided.
 
 const dataArray1 = ["array", "object", "number", "string", "Boolean"]
 const dataArray2 = ["string", "null", "Boolean", "string", "undefined"]
 // Expected output: ["array", "object", "number", "string", "Boolean", "null", "undefined"]
 
+describe('uniqueCombo', function() {
+
+   it("takes in two arrays as arguments and returns one array with no duplicate values. STRETCH: Use the spread operator to pass in a dynamic number of arguments.", () => {
+      expect(uniqueCombo(dataArray1, dataArray2)).toEqual(["array", "object", "number", "string", "Boolean", "null", "undefined"])
+    })
+  })
+
+//     ReferenceError: combined is not defined
+
+
+
+
+
 
 // b) Create the function that makes the test pass.
+
+/*
+Now for the fun part. If I'm going to tackle the stretch goal I might as well consider it from the start. 
+I'm going to combine both arrays, going to try concat for this
+Next I need to look up a method to get rid of duplicates in an array. Luckily the first link was a very good resource, you see it being used in line 189 and 191
+*/
+
+// let comboArray = () => {dataArray1.concat(dataArray2)} 
+// ^^^^ Didn't AND COULDN'T use this method when it comes to spread operator. 
+//TypeError: function is not iterable (cannot read property Symbol(Symbol.iterator))
+// at new Set (<anonymous>)
+
+
+
+comboArray = [...new Set([...dataArray1, ...dataArray2])] 
+//This is joining the arrays together 
+
+
+let uniqueCombo = () => [...new Set(comboArray)]
```

Comment:
make sure your function is const, and you pass your code in via parameters and not by calling on the variables outside the function